### DISCUSSION TITLE
SlingPlayer zip location

### DISCUSSION THREAD

**Discussion by skijohn877** [GerryDazoo/Slinger#553] [General]
Can anyone point me to the location of slingplayer zip file on github?

**Comment by regmar505** [comment]
A search under Announcement gives [https://github.com/GerryDazoo/SlingerPlayer](url).

**Comment by watanyoshi** [comment]
Open [SlingerPlayer](https://github.com/GerryDazoo/SlingerPlayer) then click <> Code, Download ZIP
